cluster-up/cluster/kind-k8s-1.23/README.md

```diff
@@ -0,0 +1,45 @@
+# K8S 1.23.3 with mdev support in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.23.3 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart.
+The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+The following needs to be executed as root.
+
+```bash
+export KUBEVIRT_PROVIDER=kind-k8s-1.23
+make cluster-up
+```
+
+The cluster can be accessed as usual:
+
+```bash
+$ cluster-up/kubectl.sh get nodes
+NAME                  STATUS   ROLES    AGE     VERSION
+kind-1.23-control-plane   Ready    master   6m14s   v1.23.3
```

Comment:
If there's only 1 node, shouldn't it also be a worker?

Reply:
Yes, it should be both master and worker node.

Reply:
But I tried to create one node cluster, and use `kubectl get nodes`. It do not show worker role.

Reply:
I assume it can't be used for much then... We probably need something like that:
https://github.com/kubevirt/kubevirtci/blob/main/cluster-up/cluster/k8s-provider-common.sh#L153
Alternatively, I guess we could enforce a minimum of 2 nodes

Reply:
I am not quite sure about it. As the taint of master node is none and master node is schedulable, I can deploy kubevirt and vmi successfully on this one node cluster. Do I still need to change the label of this node?

Reply:
Ah, got it, sounds like we're good then!